KamerConnect/Repositories/IPersonRepository.cs

```diff
@@ -10,4 +11,8 @@ public interface IPersonRepository
     void AddPasswordToPerson(Guid personId, string password, string salt);
     Person AuthenticatePerson(string email, string password);
     byte[] GetSaltFromPerson(string email);
+    void UpdatePerson(Person person);
+    void UpdatePersonality(Person person);
+    void UpdateSocial(Person person);
```

Comment:
Deze kan je ook in UpdatePerson aanroepen en private houden. Als je er toch voor kiest om ze los te houden zou ik het exacte object Personality meegeven en niet de hele Person

Reply:
zal dan sowieso Personality object meegeven, maar dezelfde comment van eerder geldt hiervoor:

"dit leek me niet handig aangezien de pagina's van die dingen aparte tabjes zijn, dus dan moet die onnodig de person updaten, en vervolgens personality."

Reply:
Helemaal goed

Reply:
heb het geprobeerd op de manier om een person mee te geven, enkel weet hij dan niet meer welke person.id bij de social of personality tabel hoort.